Lombiq.Vsix.Orchard/Services/LogWatcher/BlinkStickManager.cs

```diff
@@ -6,17 +6,15 @@
 
 namespace Lombiq.Vsix.Orchard.Services.LogWatcher
 {
-    [SuppressMessage(
-        "Critical Code Smell",
-        "S4487:Unread \"private\" fields should be removed",
-        Justification = "The _backgroundTask variable is required to keep the the Task alive.")]
     public sealed class BlinkStickManager : IBlinkStickManager
     {
         private readonly object _lock = new object();
         private BlinkStick _blinkStick;
         private bool _isInitialized;
         private CancellationTokenSource _cancellationTokenSource;
+#pragma warning disable S4487 // Unread "private" fields should be removed. This variable is required to keep the the Task alive.
```

Comment:
Pragmaknál az itt láthatóhoz hasonlóan egy megelőző sorban legyen a justification: https://github.com/Lombiq/.NET-Analyzers/blob/dev/Docs/UsingAnalyzersDuringDevelopment.md#:~:text=Example%20where%20the%20%23pragma%20suppress%20is%20suitable%20(note%20how%20the%20suppression%20only%20affects%20the%20class%20declaration%20and%20a%20justifications%20is%20provided)%3A A `disable` és `restore` sora maradjon az eredeti, csak az auto-generált commenttel.